anyTV/quasar-boilerplate
# Delta-1641 Platform > Freedom Components > use FTable from freedom-components

## Ticket references
- [Delta-1641](https://freedom.myjetbrains.com/youtrack/issue/Delta-1641)

## Summary of changes
-  use FTable from freedom-components on existing FTable demo pages


## Checklist


## Testing
- Use this configuration for package.json
```
curl https://gist.githubusercontent.com/paulyves/1882b676f65ebf57b2c54772f31ca964/raw | git apply
```
- remove `package-lock.json` and run `npm install`

Check the ff pages:
- [/table-with-tabs](https://localhost:8080/table-with-tabs)
- [/table-with-tabs-2](https://localhost:8080/table-with-tabs-2)
- [/table-without-tabs](https://localhost:8080/table-without-tabs)

PR reference:
- https://gitlab.com/anyTV/incubator/freedom-components/-/merge_requests/57

![image](https://user-images.githubusercontent.com/17078489/153821309-3c183f4e-3ddf-4d54-9ef3-9e87d5326cb4.png)
